pymodbus/repl/server/cli.py

```diff
@@ -191,12 +194,12 @@ async def interactive_shell(server):
 
 
 async def main(server):
-    with patch_stdout():
-        try:
-            await interactive_shell(server)
-        finally:
-            pass
-        warning("Bye Bye!!!")
+    # with patch_stdout():
```

Comment:
I understand most of the changes, but this one seems unrelated.

Reply:
This is required for REPL clients to work on 3.7. Not directly related.